profiles/api.py

```diff
@@ -1,20 +1,49 @@
 """Users api"""
-import re
+from collections import namedtuple
+import csv
+from datetime import datetime, timedelta
 from functools import reduce
+import logging
+import re
 import operator
 
+from django.db import transaction
 from django.db.models import Q
 from django.core.exceptions import ValidationError
 from django.core.validators import validate_email
 from django.contrib.auth import get_user_model
+from django.utils.timezone import is_naive, make_aware
 
+from klasses.models import BootcampRun, BootcampRunEnrollment
 from main.utils import first_or_none, unique, unique_ignore_case, max_or_none
-from profiles.constants import USERNAME_MAX_LEN
+from profiles.constants import (
+    ALUMS_BOOTCAMP_END_DATE,
+    ALUMS_BOOTCAMP_NAME,
+    ALUMS_BOOTCAMP_RUN_TITLE,
+    ALUMS_BOOTCAMP_START_DATE,
+    ALUMS_HEADER_FIELDS,
+    ALUMS_LEARNER_EMAIL,
+    USERNAME_MAX_LEN,
+)
+from profiles.exceptions import AlumsImportException
+from profiles.utils import usernameify
 
 User = get_user_model()
 
 CASE_INSENSITIVE_SEARCHABLE_FIELDS = {"email"}
 
+Alums = namedtuple(
```

Comment:
Since this is referring to one person, I think this should be `Alum` instead here and elsewhere